Update GSoC page with current status (#6037)
jenkins-infra/jenkins.io: content/projects/gsoc/index.adoc

== GSoC 2023

> We applied to participate in Google Summer of Code 2023. 

Reviewer: ```suggestion
We have applied to participate in Google Summer of Code 2023. 
```

Author: Thanks Kris. :+1:

Author: Thank you Kris. 👍

// * link:/projects/gsoc/2022/projects/automatic-git-cache-maintenance[Automatic git cache maintenance on the controller] by Hrushikesh Rao
// * link:/projects/gsoc/2022/projects/pipeline-step-documentation-generator[Pipeline Step Documentation Generator Improvements] by Vihaan Thora

> We are currently waiting to know whether we are accepted as a participating organisation (to be announced on Feb 22). 

Reviewer 2: ```suggestion
We are currently waiting to know wether we are accepted as a participating organisation (to be announced on Feb 22). 
```

Author: Thanks Stefan. :+1:

Author: Had a doubt on this and looked it up https://prowritingaid.com/wether-whether-weather

> We are currently waiting to know whether we are accepted as a participating organisation (to be announced on Feb 22). 

Reviewer: ```suggestion
We are currently waiting to know whether we will be accepted as a participating organisation (to be announced on Feb 23). 
```

Author: Thanks Kris. :+1:

Had definitely a doubt with "whether" and "wether" :-)

Author: Thanks for the grammatical correction, Kris.
Although a detail, I'll keep the date as is. I checked the timeline and Google will publish the participating orgs on Feb 22 at 18:00 UTC. ;-)

> We are currently waiting to know whether we will be accepted as a participating organisation (to be announced by Google on Feb 22). 

Reviewer: ```suggestion
We are currently waiting to know whether we will be accepted as a participating organisation (to be announced by Google on Feb 23). 
```

Author: This is a detail but the date is not correct 😄 : the results are announced (published) by Google on Feb 22 at 18:00 UTC. See https://developers.google.com/open-source/gsoc/timeline#february_22_-_1800_utc

Accepting the change because the PR needs to be merged.

> We are currently waiting to know whether we will be accepted as a participating organisation (to be announced by Google on Feb 23). 

Reviewer: ```suggestion
We are currently waiting to know whether we will be accepted as a participating organisation (to be announced by Google on Feb 22, 2023). 

```
